rules/0560-docker_integration_rules.xml

```diff
@@ -109,115 +109,340 @@ ID: 87900 - 87999
         <options>no_full_log</options>
     </rule>
 
-    <rule id="87914" level="3">
+    <rule id="87914" level="7">
         <if_sid>87912</if_sid>
         <field name="docker.Action">^destroy$</field>
         <description>Volume destroyed in $(docker.Actor.Attributes.driver)</description>
         <options>no_full_log</options>
     </rule>
 
     <rule id="87915" level="3">
+        <if_sid>87912</if_sid>
+        <field name="docker.Action">^mount$</field>
+        <description>Volume mounted on $(docker.Actor.Attributes.destination)</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87916" level="5">
+        <if_sid>87912</if_sid>
+        <field name="docker.Action">^unmount$</field>
+        <description>Volume unmounted from $(docker.Actor.Attributes.driver)</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87917" level="3">
         <if_sid>87900</if_sid>
         <field name="docker.status">^commit$</field>
-        <description>Container $(docker.Actor.Attributes.name) commited</description>
+        <description>Container $(docker.Actor.Attributes.name) commited an image</description>
         <options>no_full_log</options>
     </rule>
 
-    <rule id="87916" level="3">
+    <rule id="87918" level="3">
         <if_sid>87900</if_sid>
         <field name="docker.status">^tag$</field>
         <description>Image $(docker.Actor.Attributes.name) tagged</description>
         <options>no_full_log</options>
     </rule>
 
-    <rule id="87917" level="3">
+    <rule id="87919" level="3">
         <if_sid>87900</if_sid>
         <field name="docker.status">^untag$</field>
         <description>Image $(docker.Actor.Attributes.name) untagged</description>
         <options>no_full_log</options>
     </rule>
 
-    <rule id="87918" level="3">
+    <rule id="87920" level="3">
+        <if_sid>87900</if_sid>
+        <field name="docker.status">^import$</field>
+        <description>Image created from imported data</description>
+        <options>no_full_log</options>
+    </rule>
+
+    <rule id="87921" level="5">
         <if_sid>87900</if_sid>
         <field name="docker.status">^delete$</field>
         <description>Container $(docker.Actor.Attributes.name) deleted</description>
         <options>no_full_log</options>
     </rule>
 
-    <rule id="87919" level="3">
+    <rule id="87922" level="3">
         <if_sid>87900</if_sid>
         <field name="docker.status">^attach$</field>
         <description>Container $(docker.Actor.Attributes.name) attached standard input, output and error</description>
```

Comment:
`Attached local standard input, output, and error streams to container $(docker.Actor.Attributes.name)`